src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1727,14 +1722,22 @@ void TorrentHandle::appendExtensionsToIncompleteFiles()
 {
     QVector<qreal> fp = filesProgress();
     for (int i = 0; i < filesCount(); ++i) {
+        QString name = filePath(i);
         if ((fileSize(i) > 0) && (fp[i] < 1)) {
-            const QString name = filePath(i);
             if (!name.endsWith(QB_EXT)) {
                 const QString newName = name + QB_EXT;
-                qDebug("Renaming %s to %s", qPrintable(name), qPrintable(newName));
+                qDebug() << "Renaming" << name << "to" << newName;
```

Comment:
Indeed. How strange the world of Qt streams is! `endl` means flushing, `<<` adds spaces... What else do they do behind the curtain? :)